src/services/stringCompletions.ts

```diff
@@ -223,13 +223,16 @@ namespace ts.Completions.StringCompletions {
     function stringLiteralCompletionsFromProperties(type: Type | undefined): StringLiteralCompletionsFromProperties | undefined {
         return type && {
             kind: StringLiteralCompletionKind.Properties,
-            symbols: type.getApparentProperties().filter(prop =>
-                !isPrivateIdentifierPropertyDeclaration(
-                    isTransientSymbol(prop) && prop.syntheticOrigin ? prop.syntheticOrigin.valueDeclaration : prop.valueDeclaration)),
+            symbols: type.getApparentProperties().filter(isPropertyNotPrivate),

```

Comment:
While you're here, can you switch to the built-in `filter` in `core.ts` which avoids an allocation if everything is the same.

Reply:
Sure